cmd/admin-revoker/main.go

```diff
@@ -226,11 +226,14 @@ func main() {
 	case command == "auth-revoke" && len(args) == 1:
 		domain := args[0]
 		_, logger, _, sac := setupContext(c)
-		ident := core.AcmeIdentifier{Value: domain, Type: core.IdentifierDNS}
-		authsRevoked, pendingAuthsRevoked, err := sac.RevokeAuthorizationsByDomain(ctx, ident)
-		cmd.FailOnError(err, fmt.Sprintf("Failed to revoke authorizations for %s", ident.Value))
-		logger.Infof("Revoked %d pending authorizations and %d final authorizations",
-			pendingAuthsRevoked, authsRevoked)
+		req := &sapb.RevokeAuthorizationsByDomainRequest{
```

Comment:
I think for deployability we'll still need to feature flag this, if you try and use `RevokeAuthorizationsByDomain2` without the migration being applied etc it'll fail spectacularly.

Reply:
Sounds good :+1: I should have thought of that tbh.

Reply:
Resolved in b91c9da.